src/utils/prefs.js

```diff
@@ -40,4 +41,8 @@ const prefs = new PrefsHelper("devtools", {
   fileSearchRegexMatch: ["Bool", "debugger.file-search-regex-match"]
 });
 
-module.exports = { prefs };
+function setDebuggerVersion() {
+  prefs.debuggerVersion = DEBUGGER_VERSION;
+}
+
+module.exports = { prefs, setDebuggerVersion };
```

Comment:
I think it might be nice to keep the schema version private so that we only manage it here...

```js
if (prefs.debuggerPrefsSchemaVersion != prefsSchemaVersion) {
  prefs.clear()
  prefs.debuggerPrefsSchemaVersion = prefsSchemaVersion
}
```